lib/cylc/scheduler.py

```diff
@@ -56,9 +54,7 @@
     SuiteEventContext, SuiteEventError, SuiteEventHandler)
 from cylc.suite_srv_files_mgr import (
     SuiteSrvFilesManager, SuiteServiceFileError)
-from cylc.suite_status import (
-    KEY_DESCRIPTION, KEY_GROUP, KEY_META, KEY_NAME, KEY_OWNER, KEY_STATES,
-    KEY_TASKS_BY_STATE, KEY_TITLE, KEY_UPDATE_TIME, KEY_VERSION)
+from cylc.suite_status import KEY_NAME, KEY_OWNER, KEY_VERSION
```

Comment:
I'm getting the same message with my linter but it looks like it is used on line 953!

Reply:
I think the `KEY_NAME` used in [line 953](https://github.com/oliver-sanders/cylc/blob/cac70d18ff6da035cf59f2c12d83991ba9f7be2d/lib/cylc/scheduler.py#L953) is actually the property of the [suite_srv_files_mgr/SuiteSrvFilesManager class](https://github.com/oliver-sanders/cylc/blob/cac70d18ff6da035cf59f2c12d83991ba9f7be2d/lib/cylc/suite_srv_files_mgr.py#L55) ? So that import is still not used?